src/charts/gv-chart-histogram.js

```diff
@@ -48,6 +44,7 @@ export class GvChartHistogram extends ChartElement(LitElement) {
     return {
       chart: {
         type: 'column',
+        backgroundColor: '#fafafa',
```

Comment:
We should set the background to transparent and let the chart user define some CSS on the container if needed:
```suggestion
        backgroundColor: 'transparent',
```

Reply:
Done thx :)